electron-builder.js

```diff
@@ -21,18 +24,60 @@ const fs = require("fs");
  */
 
 const NIGHTLY_APP_ID = "im.riot.nightly";
-const NIGHTLY_APP_NAME = "element-desktop-nightly";
 const NIGHTLY_DEB_NAME = "element-nightly";
 
 const pkg = JSON.parse(fs.readFileSync("package.json", "utf8"));
 
+let buildMacOsUniversal = false;
+
 /**
  * @type {import('electron-builder').Configuration}
  * @see https://www.electron.build/configuration/configuration
  */
 const config = {
     appId: "im.riot.app",
     asarUnpack: "**/*.node",
+    beforePack: async (context) => {
+        if (context.electronPlatformName === "darwin" && context.arch === Arch.universal) {
+            buildMacOsUniversal = true;
+        }
+    },
+    afterPack: async (context) => {
+        if (context.electronPlatformName !== "darwin" || context.arch === Arch.universal || !buildMacOsUniversal) {
+            // Burn in electron fuses, for macOS if we are building a universal package we only need to burn fuses there
+            const ext = {
+                darwin: ".app",
+                win32: ".exe",
+                linux: [""],
```

Comment:
why is this one an array while the others are strings?

Reply:
Good question, copied it right out of the electron-builder issue example